plugins/table.md

```diff
@@ -361,7 +361,7 @@ tinymce.init({
 
 ### `table_style_by_css`
 
-This option enables you to force Table Properties dialog to use HTML5/CSS3 standards for setting cell spacing and cellpadding. By default, these are added as attributes to the table element. By setting this to true, cell spacing is applied to the table element as a `border-spacing` style and cell padding is applied to all `td` elements as a `padding` style
+This option enables you to force Table Properties dialog to use HTML5/CSS3 standards for setting cell spacing and cellpadding. By default, these are added as attributes to the table element. By setting this to true, cell spacing is applied to the table element as a `border-spacing` style and cell padding is applied to all `td` elements as a `padding` style.
```

Comment:
"force the Table Properties"?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1590/commits/c4f6e844ae7168e5343fd3e281d776e5355b09ac